pkgs/development/tools/rust/cargo-tauri/beta.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, rustPlatform
+, fetchFromGitHub
+, openssl
+, pkg-config
+, glibc
+, libsoup
+, cairo
+, gtk3
+, webkitgtk
+, darwin
+}:
+
+let
+  inherit (darwin.apple_sdk.frameworks) CoreServices Security SystemConfiguration;
+in
+rustPlatform.buildRustPackage rec {
+  pname = "tauri";
+  version = "2.0.0-beta.17";
+
+  src = fetchFromGitHub {
+    owner = "tauri-apps";
+    repo = pname;
```

Comment:
```suggestion
    repo = "tauri";
```

Reply:
The `pname is there since tauri could change names and it's used twice in src.name`